File: Software/C#/FreETarget.NET/FreETarget.NET.Data/Models/Svg/SvgElement.cs
namespace FreeTargetWeb01.Models.Svg
{
    public class SvgElement
    {
        public  SvgElementType ElementType { get; set; }
        public int X { get; set; }
        public int Y { get; set; }
        public int? R { get; set; }
        public int? Width { get; set; }
        public int ?Height { get; set; }
        public string? Fill { get; set; }
        public string? Text { get; set; }
        public int? FontSize { get; set; }
        public string? FontWeight { get; set; }
        public string? Stroke { get; set; }
        public int? StrokeWidth { get;set; }


    }
}

File: Software/BrowserClientPOC/src/svg/svgElement.ts
import { SvgElementType } from "./svgElementType";

export class SvgElement {
    elementType: SvgElementType;
    x: number;
    y: number;
    x2: number | null;
    y2: number | null;
    r: number | null;
    width: number | null;
    height: number | null;
    fill: string | null;
    text: string | null;
    fontSize: number | null;
    fontWeight: string | null;
    stroke: string | null;
    strokeWidth: number | null;
}